modules/templates/helper.go

```diff
@@ -109,6 +109,9 @@ func NewFuncMap() []template.FuncMap {
 		"CustomEmojis": func() map[string]string {
 			return setting.UI.CustomEmojisMap
 		},
+		"IsShowFullName": func() bool {
+			return setting.UI.DefaultShowFullName
+		},
```

Comment:
There are a lot of `setting.UI.xxx` in helper.go

I guess in the future (next PR), we can just do `"SettingUI": &setting.UI` to expose all UI settings to template.